dvc/logger.py

```diff
@@ -10,6 +10,11 @@
 import logging.config
 import colorama
 
+try:  # https://docs.python.org/3/library/exceptions.html#RecursionError
+    RecursionError
+except NameError:  # Python < 3.5
+    RecursionError = RuntimeError
```

Comment:
How about we move it to `dvc/utils/compat.py`, so it is easier to get rid of it next year when py35 is dead?